rclrs/src/qos.rs

```diff
@@ -179,6 +179,12 @@ pub struct QoSProfile {
     pub avoid_ros_namespace_conventions: bool,
 }
 
+impl Default for QoSProfile {
+    fn default() -> Self {
```

Comment:
Should we make a note here that this is setting the QOS profile to the RCL default? Just to avoid any confusion?

Reply:
Done! https://github.com/ros2-rust/ros2_rust/pull/361/commits/6ac6caa3655a4458739d7d1457c20200bfa627ca

Reply:
@luca-della-vedova thanks 🙂